example/utils/index.ts

```diff
@@ -33,6 +33,10 @@ export type State = {
   flatTextSecureEntry: boolean;
   outlineTextSecureEntry: boolean;
   iconsColor: IconsColor;
+  flatMultilineCustomHeightNoLabel: string;
+  flatMultilineCustomHeightNoLabelTop: string;
+  outlinedMultilineCustomHeightNoLabel: string;
+  outlinedMultilineCustomHeightNoLabelTop: string;
```

Comment:
Did you intend to add an example for this one as well? Only seems to have default value but not connected to an input.

Reply:
Removed